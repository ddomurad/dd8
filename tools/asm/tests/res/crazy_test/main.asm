.inc "defs.asm"
.inc "test_lib.asm"

.org 0x1000
ldai 0x72
sta REG1
ldai 0x1c 
sta REG2
jsr simple_add
