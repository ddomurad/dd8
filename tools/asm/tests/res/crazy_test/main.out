:070F0000A5006501850260F8
:0B100000A9728500A91C850120000FCB
:00000001FF
